### PR TITLE
Get up and go step creation

### PR DESCRIPTION
**Pull request by jenlmoore**
# *Create Full Get Up And Go Step*

## :recycle: Current situation & Problem
need q11

## :bulb: Proposed solution
add q11 with custom step

### Related PRs
*Reference the related PRs*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: Utah.xcodeproj/xcshareddata/xcschemes/Utah.xcscheme [hunk 1]
```diff
@@ -71,7 +71,7 @@
          <BuildableReference
             BuildableIdentifier = "primary"
             BlueprintIdentifier = "653A254C283387FE005D4D48"
-            BuildableName = "U-Step.app"
+            BuildableName = "Utah.app"
```
Comment (BKrajancich) [inline Utah.xcodeproj/xcshareddata/xcschemes/Utah.xcscheme]: Not sure why this changed?

File: UtahModules/Sources/UtahSchedule/Edmonton/EdmontonViewController.swift [hunk 2]
```diff
@@ -154,70 +155,19 @@ struct EdmontonViewController: UIViewControllerRepresentable {
...
-                    recorderConfigurations.append(ORKDeviceMotionRecorderConfiguration(identifier: "getupAndGo-deviceMotion", frequency: 100))
-                    step.recorderConfigurations = recorderConfigurations
-                    return step
-            }
+            // Walk
+            // Instruction step
+            let getUpIntroStep = ORKInstructionStep(identifier: "GetUpAndGoIntro")
+            getUpIntroStep.title = "Get Up and Go"
+            getUpIntroStep.text = """
+            Sit in this chair with your back and arms resting. Then, when you are ready, click the ‘Next’ button and please stand up and walk at a safe and comfortable pace approximately 3m away. Then click the 'Stop' button. When you are finished, turn around and return to the chair and sit down.
```
Comment (BKrajancich) [inline UtahModules/Sources/UtahSchedule/Edmonton/EdmontonViewController.swift]: Update this to that given by the mentors

**Review by BKrajancich — CHANGES_REQUESTED** [review]
Minor comments. Impressive work on the timer functionality!

**Review by BKrajancich — APPROVED** [review]

File: Utah.xcodeproj/xcshareddata/xcschemes/Utah.xcscheme (hunk 1 above)
Comment (jenlmoore) [reply Utah.xcodeproj/xcshareddata/xcschemes/Utah.xcscheme]: Hmm Im not sure either, shoot. Were you able to resolve this?

Comment (BKrajancich) [reply Utah.xcodeproj/xcshareddata/xcschemes/Utah.xcscheme]: Nah, still working on it

**Pull request merged**